pkg/controller/installation/windows_controller.go

```diff
@@ -0,0 +1,597 @@
+	// and also makes sure we spot when things change that might not trigger a reconciliation.
+	err = utils.AddPeriodicReconcile(c, reconcilePeriod)
+	if err != nil {
+		return fmt.Errorf("tigera-windows-controller failed to create periodic reconcile watch: %w", err)
+	}
+
+	return nil
+}
+
+type ReconcileWindows struct {
+	config               *rest.Config
+	client               client.Client
+	scheme               *runtime.Scheme
+	watches              map[runtime.Object]struct{}
+	autoDetectedProvider operatorv1.Provider
+	status               status.StatusManager
+	enterpriseCRDsExist  bool
+	amazonCRDExists      bool
+	clusterDomain        string
+	manageCRDs           bool
+}
+
+// newWindowsReconciler returns a new reconcile.Reconciler
+func newWindowsReconciler(mgr manager.Manager, opts options.AddOptions) (*ReconcileWindows, error) {
+	statusManager := status.New(mgr.GetClient(), "calico-windows", opts.KubernetesVersion)
+
+	r := &ReconcileWindows{
+		config:               mgr.GetConfig(),
+		client:               mgr.GetClient(),
+		scheme:               mgr.GetScheme(),
+		watches:              make(map[runtime.Object]struct{}),
+		autoDetectedProvider: opts.DetectedProvider,
+		status:               statusManager,
+		amazonCRDExists:      opts.AmazonCRDExists,
+		enterpriseCRDsExist:  opts.EnterpriseCRDExists,
+		clusterDomain:        opts.ClusterDomain,
+		manageCRDs:           opts.ManageCRDs,
+	}
+	r.status.Run(opts.ShutdownContext)
+	return r, nil
+}
+
+// Reconcile reads that state of the cluster for a Installation object and makes changes based on the state read
+// and what is in the Installation.Spec. The Controller will requeue the Request to be processed again if the returned error is non-nil or
+// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
+func (r *ReconcileWindows) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+	reqLogger := logw.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	reqLogger.Info("Reconciling Installation.operator.tigera.io")
+
+	// Get the installation object if it exists so that we can save the original
+	// status before we merge/fill that object with other values.
+	instance := &operatorv1.Installation{}
+	if err := r.client.Get(ctx, utils.DefaultInstanceKey, instance); err != nil {
+		if apierrors.IsNotFound(err) {
+			reqLogger.Info("Installation config not found")
+			r.status.OnCRNotFound()
+			return reconcile.Result{}, nil
+		}
+		reqLogger.Error(err, "An error occurred when querying the Installation resource")
+		return reconcile.Result{}, err
+	}
+
+	preDefaultPatchFrom := client.MergeFrom(instance.DeepCopy())
+
+	// Mark CR found so we can report converter problems via tigerastatus
+	r.status.OnCRFound()
+	// SetMetaData in the TigeraStatus such as observedGenerations.
+	defer r.status.SetMetaData(&instance.ObjectMeta)
+
+	// Changes for updating Installation status conditions.
```

Comment:
This logic is going to cause problems because we have two controllers fighting over the status of the installation. I think we need logic to merge the conditions from both controllers.

For now, we can probably get away with just remove this block from this controller. But we should track fixing this logic in a future PR.